### DISCUSSION TITLE
Where are files stored in the browser?

### DISCUSSION THREAD

**Discussion by Heimdallr01** [TransformerOptimus/SuperAGI#665] [Q&A]
When using uploaded files, SuperAGI always says it can't find any files. Do I need to tell it a path to the file?

> Tool Read File returned: Error1: File 'xxx.txt' not found., FileNotFoundError, args: {'file_name': 'xxx.txt'}

**Comment by neelayan7** [comment]
In the Resource Manager. 
Read File works by specifying the exact name of the file you uploaded.
